graylog2-web-interface/src/components/inputs/InputsList.jsx

```diff
@@ -83,15 +136,28 @@ const InputsList = createReactClass({
         </IfPermitted>
         }
 
+        <Row id="filter-input" className="content input-new">
+          <Col>
+            <SearchForm onSearch={this._onFilterInputs}
+                        label="Filter inputs"
+                        topMargin={0}
+                        onReset={this._onFilterReset}
+                        wrapperClass={InputsListStyle.filterWrapper}
+                        searchButtonLabel="Filter"
+                        placeholder="Search by title"
```

Comment:
I think having an input above without any label and another one below it with a label looks kind of funny. Maybe should we try to make both inputs look similar? This is a quick attempt:

Before:
<img width="1792" alt="before" src="https://user-images.githubusercontent.com/716185/40480350-7859d94e-5f4e-11e8-87e8-4eb6cf66df73.png">

After:
<img width="1792" alt="after" src="https://user-images.githubusercontent.com/716185/40480349-783b7558-5f4e-11e8-8072-1ff06d3e2bb9.png">

Reply:
In any case we should only use one verb in the `SearchForm`, either "search" or "filter".